extensions/amp-jwplayer/1.0/amp-jwplayer.js

```diff
@@ -0,0 +1,117 @@
+import {getDataParamsFromAttributes} from '#core/dom';
+import {dict} from '#core/types/object';
+
+import {isExperimentOn} from '#experiments';
+
+import {userAssert} from '#utils/log';
+
+import {BaseElement} from './base-element';
+
+import {CSS} from '../../../build/amp-jwplayer-1.0.css';
```

Comment:
I don't think this CSS needs to be registered in the AMP layer, since the AMP boilerplate accounts for pre-upgrade styles already, and jwplayer doesn't have non-pre-upgrade styles.